src/mocks/artifact.ts

```diff
@@ -0,0 +1,40 @@
+import { Artifact, ArtifactType } from '@/models'
+import { MockFunction } from '@/services/Mocker'
+import { choice } from '@/utilities'
+
+export const randomArtifact: MockFunction<Artifact, [Partial<Artifact>?]> = function(overrides = {}) {
+  const type = choice<ArtifactType>([ArtifactType.Markdown, ArtifactType.Result, ArtifactType.Table])
+  let data = null
+
+  switch (type) {
+    case ArtifactType.Result:
+      data = {
+        type: choice(['literal']),
```

Comment:
choice from a single value?

Reply:
it's the illusion of choice, really

Reply:
for real though I've got some more updates to this and there will be an actual choice when I write real mocks for result and table artifacts data - can ignore for now